programs/ssl/ssl_server2.c

```diff
@@ -2960,8 +3060,16 @@ int main( int argc, char *argv[] )
 
 #if defined(MBEDTLS_SSL_EXPORT_KEYS)
     if( opt.eap_tls != 0 )
+    {
         mbedtls_ssl_conf_export_keys_ext_cb( &conf, eap_tls_key_derivation,
                                              &eap_tls_keying );
+    }
+    else if( opt.nss_keylog != 0 )
```

Comment:
I'm not sure, how important it is for this particular program, but if we enable `eap_tls`, the new feature will not work without any heads-up for the user.

Reply:
Good point, I've pushed a commit to address this.